tests/integration/privateweave/kubernetes.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_autoscaling_group" "bastion-privateweave-example-com" {
   launch_configuration = "${aws_launch_configuration.bastion-privateweave-example-com.id}"
   max_size = 1
   min_size = 1
-  vpc_zone_identifier = ["${aws_subnet.private-us-test-1a-privateweave-example-com.id}"]
+  vpc_zone_identifier = ["${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"]
```

Comment:
@justinsb 

We aren't renaming `private` subnets to `utility` subnets are we?

In general the 2 conventions that we were using

# Public
These are where the NAT gateways live, and have the synonym **utility**

# Private
These are private, and hold the resources like instances